src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax;
+
+import static core.basesyntax.ColorSupplier.getRandomColor;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        Ball randomBall = new Ball();
+        ColorSupplier.Colors randomColor = ColorSupplier.Colors.valueOf(getRandomColor());
+
+        randomBall.setColor(randomColor.toString());
+
+        randomBall.setNumber(new Random().nextInt(100) + 1);
```

Comment:
create a constant from 100
For example: private static final String MY_CONSTANT = "hello";